src/app/WriteHandler.h

```diff
@@ -69,9 +69,7 @@ class WriteHandler
      *  @retval #Others If fails to process read request
      *  @retval #CHIP_NO_ERROR On success.
      */
-    CHIP_ERROR OnWriteRequest(Messaging::ExchangeContext * apExchangeContext, System::PacketBufferHandle aPayload);
-
-    CHIP_ERROR SendWriteResponse(System::PacketBufferHandle aPayload);
+    CHIP_ERROR OnWriteRequest(Messaging::ExchangeContext * apExchangeContext, System::PacketBufferHandle && aPayload);
```

Comment:
Appears to be a logic change in here?

Reply:
Our general convention now seems to be to pass packetbuffers by rvalue reference, I intend to use rvalue, but i make a typo here which use copy constructor, functionality is same, copy constructor make additional memory copy. Here I correct it to use rvalue referance.